src/main.rs

```diff
@@ -33,16 +37,15 @@ fn main() {
       .lint(file_name, source_code, rules)
       .expect("Failed to lint");
 
-    diagnostics.extend(file_diagnostics)
-  }
-
-  if !diagnostics.is_empty() {
-    for d in diagnostics.iter() {
-      eprintln!(
-        "error: {} ({}) at {}:{}:{}",
-        d.message, d.code, d.location.filename, d.location.line, d.location.col
-      );
+    error_counts += file_diagnostics.len();
+    if !file_diagnostics.is_empty() {
+      report_util::report_filename(&file_diagnostics[0].location.filename);
+      for d in file_diagnostics.iter() {
+        report_util::report_line_src(d.location.line, &d.line_src);
+        report_util::place_glyph(d.location.line, d.location.col);
+        report_util::report_error(&d.code, &d.message);
+      }
```

Comment:
This is something that bugs me about ESLint's output. Something like
```
../deno/std/http/server.ts =>
  93|       } catch {}
                    ^
  (noEmpty) Empty block statement
```
doesn't have anything to ctrl + click in an integrated terminal to take you straight to the offending line.
If there is an error at `std/http/server.ts`, line 93, column 16, it's conventional to output the following adjacently: `std/http/server.ts:93:16`. Clippy and other Rust errors do this correctly:
```
error: unused import: `std::path::Path`
  --> cli/main.rs:96:5
   |
96 | use std::path::Path;
   |     ^^^^^^^^^^^^^^^
   |
```

Reply:
@nayeemrmn I agree, that's been bugging me as well - having a way to link to problematic location is great. It seems like Rust's output gets it right; @disizali sorry to bother, but maybe you could try above format?

Reply:
It's my pleasure, I will definitely do it.
I think rust output is really telling too,
I try to be like that